src/components/FilterBar/FilterDisplay/Inputs/SelectInput.react.js

```diff
@@ -1,8 +1,16 @@
 export class SelectInput extends React.Component {
-  constructor(props) {
-    super(props);
+  constructor(props, context) {
```

Comment:
Interesting, I hadn't observed that. I assumed it would call the constructer whenever a new instance of the class was instantiated. Where can I read about this?